### PR TITLE
Fix lab08 ex03 skel file

### PR DESCRIPTION
**Pull request by lucigrigo**

### CODE REVIEW

File: labs/lab08/skel/lin/3-safety/critical.c [hunk 1]
```diff
@@ -51,7 +51,7 @@ void acquire_lock(void)
 		 * If you're interested in knowing more, see this link:
 		 * https://www.internalpointers.com/post/understanding-memory-ordering.
 		 */
-    }
+	}
```
Comment (lucigrigo) [inline labs/lab08/skel/lin/3-safety/critical.c:55]: I think here it was indented with spaces, not tabs, not sure tho.

File: labs/lab08/skel/lin/3-safety/critical.c [hunk 2]
```diff
@@ -27,7 +27,7 @@ void acquire_lock(void)
 #ifdef USE_SPINLOCK
 	/* Busy waiting */
 	while (1) {
-		/* TODO2: Implement spinlock lock method
+		/** TODO2: Implement spinlock lock method
```
Comment (teodutu) [inline labs/lab08/skel/lin/3-safety/critical.c]:
```suggestion
		/**
		 * TODO2: Implement spinlock lock method
```

File: labs/lab08/skel/lin/3-safety/critical.c [hunk 3]
```diff
@@ -60,7 +60,15 @@ void acquire_lock(void)
 void release_lock(void)
 {
 #ifdef USE_SPINLOCK
-
+		/** TODO2: Implement spinlock release method
```
Comment (teodutu) [inline labs/lab08/skel/lin/3-safety/critical.c]:
```suggestion
		/**
		 * TODO2: Implement spinlock release method
```

File: labs/lab08/sol/lin/3-safety/critical.c [hunk 4]
```diff
@@ -27,7 +27,7 @@ void acquire_lock(void)
 #ifdef USE_SPINLOCK
 	/* Busy waiting */
 	while (1) {
-		/* TODO2: Implement spinlock lock method
+		/** TODO2: Implement spinlock lock method
```
Comment (teodutu) [inline labs/lab08/sol/lin/3-safety/critical.c]:
```suggestion
		/**
		 * TODO2: Implement spinlock lock method
```

File: labs/lab08/sol/lin/3-safety/critical.c [hunk 5]
```diff
@@ -40,7 +40,7 @@ void acquire_lock(void)
 		if(__atomic_test_and_set(&lock, __ATOMIC_SEQ_CST) == false)
 			return;
 
-		/* Short explanation: High-level language code is translated into
+		/** Short explanation: High-level language code is translated into
```
Comment (teodutu) [inline labs/lab08/sol/lin/3-safety/critical.c]:
```suggestion
		/**
		 * Short explanation: High-level language code is translated into
```

File: labs/lab08/sol/lin/3-safety/critical.c [hunk 6]
```diff
@@ -62,7 +62,7 @@ void acquire_lock(void)
 void release_lock(void)
 {
 #ifdef USE_SPINLOCK
-		/* TODO2: Implement spinlock release method
+		/** TODO2: Implement spinlock release method
```
Comment (teodutu) [inline labs/lab08/sol/lin/3-safety/critical.c]:
```suggestion
		/**
		 * TODO2: Implement spinlock release method
```

**Review by teodutu — CHANGES_REQUESTED** [review]
Format multiline comments like this:
```
/**
 * comment
 */
```
and it should be fine.

**Review by teodutu — APPROVED** [review]
I think it's good now. Squash the commits into a single one and the PR should be fine.

**Pull request merged**